ChangeLog

```diff
@@ -1,3 +1,45 @@
+# 2024-07-31 [version 1.14.0]
+## 🐣 New Features
+* Upgrade Java version to 8.
+Read and return null if cache server stored NullValue instance.
+* Provide shutdown(long, TimeUnit) method for ArcusClient.
+* TranscoderService is removed. Decoding value by asyncGet/asyncGetBulk will be done in user thread (like tomcat thread).
+  BulkService properties in ConnectionFactoryBuilder is not available.
+* bulk methods using varargs keys are deprecated. Instead, methods using collection keys are still available.
+* Add auto import setting.
+* Change the way ArcusClient is named.
+* Add DNS cache TTL verification.
+* Add ArcusClient factory methods without ConnectionFactoryBuilder.
+* ArcusClient factory methods without zookeeper address are deprecated.
+* ADD async mop upsert method.
+* Remove unused TimeoutException constructors.
+
+## 🔧 Enhancements
+* Refactor redundant concurrent data structure, unnecessary modifier
```

Comment:
`data structure, unnecessary modifier -> data structure and unnecessary modifier.`